tasks/php_apache.yml

```diff
@@ -73,6 +74,12 @@
 
   - block:
 
+    - name: LAMP | FAST-CGI configuration for php
+      template: src={{role_dir}}/templates/apache2/conf/fastcgi-php7.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf
```

Comment:
This part needs also be moved above BEFORE fastcgi module is enabled.

Also, I (and potentially some other guys) have this role used in quite older deployments, thus we need to support backward compability.

All parts, that refer to php7, need to be aligned by php_family_prefix (i.e. php_family_prefix == php7).

Also original lines 1-5 need to be reverted back
(template: src={{role_dir}}/templates/apache2/conf/fastcgi.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf)

but this time in when they need to match to php_family_prefix php5